docs/evaluation/load_test.Rmd

```diff
@@ -0,0 +1,756 @@
+  gen_steps_df$step<-factor(gen_steps_df$step, levels = unique(gen_steps_df$step))
+  
+  labels<-c("[U] Session start", "[U] Login pause","[U] Pause before upload", "[o2r] ERC Upload","[U] Metadata modification","[o2r] ERC publication","[U] Execution pause", "[o2r] Job execution")
+  colors<-c("#eceff1","#cfd8dc","#b0bec5","#187de9","#90a4ae","#0c56bc","#78909c","#00308f")
+  if(length(unique(gen_steps_df$step))!=8){
+    labels<-c("[U] Session start", "[U] Login pause", "[U] Execution pause","[o2r] Job execution" )
+    colors<-c("#eceff1","#cfd8dc","#78909c","#00308f")
+  }
+  
+  ggplot(gen_steps_df) + geom_segment(aes(x=start,xend=end, y=job_id, yend=job_id, color=step),size=5)+scale_color_manual(labels =labels, values=colors)+ggtitle(paste0("Timeline"))+xlab("Time (s)")+ylab("Job Execution ID")+labs(color="User steps", caption=paste0("Test ID: ",test_id))+theme(legend.position="bottom",legend.box="vertical",legend.title=element_text(size=8),legend.text = element_text(size = 7),legend.key.size = unit(1,"line"),panel.background = element_blank(),panel.grid=element_line(colour="black", size=0.1), panel.border = element_rect(colour = "black", fill=NA, size=0.1),legend.key=element_blank())+facet_grid(rows = vars(compendium_id),scales="free")+theme(strip.text.y = element_text(angle = 0))
+}
+
+```
+
+# Step times
+
+This functions creates a plot using the synchronized times of the test representing the api steps and the job execution steps for creation or examination sessions.
+
+```{r}
+
+plot_step_times<-function(df_report,test_id){
+  
+  df_report$duration<-df_report$end-df_report$start
+  
+  duration_df<-df_report[df_report$step_type!="general_step",c(1,2,3,8)]
+  duration_df<-duration_df%>%pivot_wider(names_from = step, values_from = duration)
+  num_col<-ncol(duration_df)
+  duration_df[is.na(duration_df)] <- 0
+  duration_df <- duration_df %>% mutate(across(c(compendium_id:job_id),factor))
+  names<-names(duration_df)[c(-1,-2)]
+  
+  labels<-c("[API] Upload","[API] Metadata \n request","[API] Metadata\n upload","[API] Job Execution \n request", "[JOB] Status","[JOB] Validate bag","[JOB] Generate \n configuration", "[JOB] Validate \n compendium","[JOB] Generate \n manifest","[JOB] Image prepare","[JOB] Image build","[JOB] Image execute", "[JOB] Check", "[JOB] Image save", "[JOB] Cleanup")
+  
+  if(num_col==14){
+    labels<-labels[c(-1:-3)]
+  }
+  
+  nr.cols<-length(unique(duration_df$compendium_id))
+  session_colors<-colorRampPalette(brewer.pal(9, "Paired"))(nr.cols)
+  
+  ggparcoord(duration_df,columns = 3:num_col, groupColumn = 1, showPoints = TRUE, scale="globalminmax") + scale_color_manual(values=session_colors)+ theme(axis.text.x = element_text(size=8,angle = 45,hjust=1)) + labs(x = "Steps",y="Seconds (s)",title="o2R API processing times", caption = paste0("Test ID: ",test_id), col="Compendium ID") + geom_line(size=1)+ scale_x_discrete(breaks=names,labels=labels)
+}
+
+```
+
+
+##### Test execution
```

Comment:
The previous headlines is online level `#`. Please make sure the headlines use semantically meaningful levels.